roles/lib_dyn/tasks/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+# tasks file for lib_dyn
```

Comment:
I'd prefer to have this as a library only and create another role that does what you mentioned.  That way we can push this upstream.